doc/user/balance.md

```diff
@@ -0,0 +1,9 @@
+## Scale storage nodes and Balance
+
+Scaling out Storage is divided into two stages. In the first stage, you need to wait for the status of all newly created Pods to be Ready. 
+
+In the second stage, the BALANCE DATA and BALANCE LEADER command is executed. We provide a parameter `enableAutoBalance` in crd to control whether to automatically balance data and leader.
```

Comment:
Adjust the structure?

For example:

Scaling out Storage is ... 

- In the first stage, ...
- In the second stage, ...

We provide ...

Reply:
It's need to update `config/samples/apps_v1alpha1_nebulacluster.yaml` file ? And helm charts?

Reply:
I will update the apps_v1alpha1_nebulacluster.yaml in another pr